satellite/satellitedb/migrate.go

```diff
@@ -815,6 +815,20 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					);`,
 				},
 			},
+			{
+				Description: "Change type of disqualified column of nodes table to timestamp",
+				Version:     32,
+				Action: migrate.SQL{
+					`ALTER TABLE nodes
+						ALTER COLUMN disqualified DROP DEFAULT,
+						ALTER COLUMN disqualified DROP NOT NULL,
+						ALTER COLUMN disqualified TYPE timestamp with time zone USING
+							CASE disqualified
+								WHEN true THEN TIMESTAMP WITH TIME ZONE '2019-06-15 00:00:00+00'
```

Comment:
is it ok for us to hardcode this specific date in? Does it matter if it is not the correct date right now?

Reply:
I should have written my reason in the commit message or in a comment here.

At the beginning I had a `now()` call for setting the date, however I thought that a fix date would be better because if there is a bug somewhere (in this migration or even there are already nodes in the DB with `disqualified = FALSE`) we could easily identify the nodes which where disqualified by this migration.

The date could be any in the past, but I applied my random logic thinking in one that it isn't too old but it's before the release of this and used the 15th because it was exactly the half of the month.

Reply:
That makes sense. Thank you for the explanation :)